vue/sbc-common-components/src/components/BaseAddress.vue

```diff
@@ -37,7 +37,8 @@
           <!-- NB1: AddressComplete needs to be enabled each time user clicks in this search field.
                NB2: Only process first keypress -- assumes if user moves between instances of this
                    component then they are using the mouse (and thus, clicking). -->
-          <v-text-field autocomplete="off"
+          <v-text-field autocomplete="chrome-off"
+                        :name="Math.random()"
```

Comment:
Math.random will update in every change .  can we use string

Reply:
The point is to have this name be different every time (to prevent Chrome from remembering and auto-filling this field).